run-tests.sh

```diff
@@ -23,7 +23,7 @@
 pydocstyle reana_job_controller && \
 isort -rc -c -df **/*.py && \
 check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
+sphinx-build -qnN docs docs/_build/html && \
```

Comment:
It might be good to generate a "temporary" OpenAPI specs and compare it with previously-generated and committed one, so that developers are alerted if they changed something important and need to regenerate it.